keel-core/src/main/kotlin/com/netflix/spinnaker/keel/notifications/friendlyTime.kt

```diff
@@ -0,0 +1,29 @@
+package com.netflix.spinnaker.keel.notifications
+
+import java.time.Duration
+
```

Comment:
I worked hard on this and am keeping it in the code base even though I'm not using it yet. YET.

Reply:
i feel this to my core.

Reply:
omg I sware I saw this as an interview question once 😃